### PR TITLE
Scholarship page updates for 2021 Season

### PR DESCRIPTION
**Pull request by quantumfusion**

### CODE REVIEW

**Review by scottnuma — APPROVED** [review]

File: programs/scholarship.md [hunk 1]
```diff
@@ -5,28 +5,32 @@ hero-message: Pioneers in Engineering Alumni Scholarship
...
+  <a href="https://scholarship.pierobotics.org/accounts/google/login" class="btn btn-primary">Go to the web application! (Sign in with Google)</a>
+</center>
 
 ### Resources
-The following documents from the 2020 season are provided for your convenience.
+The following documents from the 2021 season are provided for your convenience.
 
-* [2020 Scholarship Workshop Slides](https://docs.google.com/presentation/d/1J-z0iznrPlQMSGBEmLeFR1X9qJ8um8HZVstEEf3SVuA/edit?usp=sharing)
-* 2020 Application Summary (to be released)
+* 2021 Scholarship Workshop Slides (TBD)
```
Comment (dwai213) [inline programs/scholarship.md:32]: You can use this link

https://docs.google.com/presentation/d/1EyMoDBchmMiFWUELNvXsOCbYNwaNxYieCoZPkqjhUNc/edit#slide=id.p

File: programs/scholarship.md [hunk 2]
```diff
@@ -5,28 +5,32 @@ hero-message: Pioneers in Engineering Alumni Scholarship
 ---
 
 # Pioneers in Engineering Alumni Scholarship
-Pioneers in Engineering strives to provide a quality STEM educational experience for Bay Area high school students. In keeping with our mission to promote science and engineering, we are happy to announce our seventh annual PiE alumni-funded scholarship for PiE students. The scholarship will be for a one-time amount of $2000.
+Pioneers in Engineering strives to provide a quality STEM educational experience for Bay Area high school students. In keeping with our mission to promote science and engineering, we are happy to announce our eighth annual PiE alumni-funded scholarship for PiE students. The scholarship will be for a one-time amount of $2000.
```
Comment (dwai213) [inline programs/scholarship.md:8]: Based on this, https://pierobotics.slack.com/archives/CJ82STX89/p1614875922055300

Let's go mention that finalists will get $500 this year

**Review by dwai213 — CHANGES_REQUESTED** [review]

File: programs/scholarship.md (hunk 2 above)
Comment (quantumfusion) [reply programs/scholarship.md:8]: Has the other docs been updated?

Comment (quantumfusion) [reply programs/scholarship.md:8]: I would like to hold off on making this change. We need to update a few other places.

**Pull request merged**